packages/pci-sdk-web/src/index.ts

```diff
@@ -181,7 +182,7 @@ function _initIframe(
 	pciElement: HTMLElement | null = document.getElementById('apto-pci-sdk'),
 	size?: Size,
 	values?: Values,
-	theme = '1',
+	theme = 'light',
```

Comment:
I think we shouldn't set the default theme here, this should be optional. All the default values are set in the iFrame. If the user doesn’t specify any theme, we shouldn’t pass this parameter to the iFrame.